src/WaxSigningApi.ts

```diff
@@ -184,7 +184,13 @@ export class WaxSigningApi {
   }
 
   private async receiveLogin(event: { data: any }): Promise<boolean> {
-    const { verified, userAccount, pubKeys, whitelistedContracts } = event.data;
+    const {
+      verified,
+      userAccount,
+      pubKeys,
+      whitelistedContracts,
+      isTemp
```

Comment:
what does `isTemp` mean?

Reply:
ohhhh I see. Does this mean an account with no blockchain account yet?

Reply:
yes that is what the isTemp, isTemp is that you dont have a block chain account yet